meinberlin/apps/cms/templates/meinberlin_cms/blocks/docs_block.html

```diff
@@ -1,9 +1,12 @@
 {% load wagtailimages_tags wagtailcore_tags %}
+{% load wagtailcore_tags block_tags %}
```

Comment:
you can and should add all the templatetags to one line. So ehre you just add block _tags to the first line between the brackets and all template tags are loaded.